sdk/servicebus/azure-mgmt-servicebus/README.md

```diff
@@ -1,12 +1,13 @@
 # Microsoft Azure SDK for Python
```

Comment:
It looks like you made changes to the `azure-mgmt-servicebus` library, which is maintained by a separate team.

If you would like to contribute to the mgmt plane SDKs, please open a new PR and we can tag the team in charge of `azure-mgmt-servicebus` for a review.